bogota-abril-2019/css-magazine-cover/david-villamizar/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8" />
+    <title>Wired UK Amazon's Next Move</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link rel="manifest" href="site.webmanifest" />
+    <link rel="apple-touch-icon" href="icon.png" />
+    <!-- Place favicon.ico in the root directory -->
+
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/main.css" />
+
+    <meta name="theme-color" content="#fafafa" />
+  </head>
+
+  <body>
+    <!--[if IE]>
+      <p class="browserupgrade">
+        You are using an <strong>outdated</strong> browser. Please
+        <a href="https://browsehappy.com/">upgrade your browser</a> to improve
+        your experience and security.
+      </p>
+    <![endif]-->
+
+    <!-- Add your site or application content here -->
+    <div id="magazine-cover">
+      <img id="fake-jeff" src="img/fake-bezos.jpg" alt="Fake Jeff Bezos" />
+      <h1 id="magazine-title">
+        <img src="img/wired-logo.svg" alt="Wired Logo" />
+        <span>
+          UK Edition
+        </span>
+      </h1>
+      <h2 id="issue-title">Amazon's Next Move</h2>
+      <section>
+        <h3>Exclusive</h3>
+        <h4>From Alexa to Drones:</h4>
+        <h5>The Jeff Bezos Master Plan</h5>
+      </section>
+      <section>
+        <span>Plus</span>
```

Comment:
I see you are using here a <span> but in other sections, you are using <h3> or <h4>. Be careful with using <h3> or <h4> because of size instead of semantic.

Reply:
Here the `<span>` is the small yellow text that's part of the main title of the cover, hence inside the same `<h1>`, but it's shown in a very different manner so I put it in a `<span>` to apply special styling to it. Is there a better way to do this? I did it too for the `<span>` that represents the 'plus' above the lower right headings.

![image](https://user-images.githubusercontent.com/4100566/56929286-d3111580-6a9e-11e9-9291-9653e3b1914a.png)